sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample_BuildModel.md

```diff
@@ -0,0 +1,59 @@
+# Build a model
+
+This sample demonstrates how to build a model with your own data. A custom model can output structured data that includes the relationships in the original document. After you build the model, you can test and retrain it and eventually use it to reliably extract data from more documents according to your needs.
+
+Please note that models can also be created using a graphical user interface such as the [Form Recognizer Labeling Tool][labeling_tool].
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `DocumentModelAdministrationClient`
+
+To create a new `DocumentModelAdministrationClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateDocumentModelAdministrationClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new DocumentModelAdministrationClient(new Uri(endpoint), credential);
+```
+
+## Build a model
+
+Build custom models to extract text, field values, selection marks, and table data from documents. Custom models are trained with your own data, so they're tailored to your documents.
```

Comment:
extract -> analyze?

Reply:
I copied this from the feedback Paul gave Python on the custom models part. So I prefer to leave as is